src/Simplic.OxS.GrpcSample.Server/Protos/user.proto

```
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+option csharp_namespace = "Simplic.OxS.GrpcSample.Server.Protos";
+
+package user;
```

Buf (1.61.0)

Files with package "user" must be within a directory "user" relative to root but were in directory "src/Simplic.OxS.GrpcSample.Server/Protos".

(PACKAGE_DIRECTORY_MATCH)